### ISSUE TITLE
cant reproduce results

### ISSUE THREAD

**Issue by kk1153** [F-Salehi/CURE_robustness#1]
results cant be reproduced according to the paper.
Have you been able to do that?

**Comment by F-Salehi** [comment]
I did a while ago, but I think I was not using the same hyper-parameters as in the paper. 
I used the regularizer \lambda larger than what mentioned in the paper, and at the end of the training, I used few epochs using SGD instead of Adam.

**Comment by wahrheit-git** [comment]
ok. can you share with me the parameters you used such as 
h and \lambda
num_epochs
learning rate

**Comment by averyma** [comment]
Hi, I am also hoping to know if it would be possible for the author to share the hyperparams and optimization settings that were used to produce the results on CIFAR10 with resnet18?
